public/index.html

```diff
@@ -7,7 +7,6 @@
     window.indexedDB;
   </script>
   <meta content="minimum-scale=1, initial-scale=1, maximum-scale=1, width=device-width, shrink-to-fit=no" name="viewport">
-  <link rel="stylesheet" href="/static/css/tabler-icons.min.css">
```

Comment:
In this file and in resources/index.html, should references to js/tabler-icons-react.min.js and js/prop-types.min.js be removed?

Reply:
The css file was included in our own bundle [here](https://github.com/logseq/logseq/pull/11468/files#diff-68779516c684641c27f15cca7118225669038e9361d6dd94fa47a05c4f1a3ac8R27). That's not the case for javascript resources. I guess there is a broader issue, which is how and when should our resources be loaded. Should we use `frontend.util/js-load`? Using the `deref` helps speeding up the process, so the potential gains might be marginal. In any case, I wanted to focus on removing dist resources from the repo in this PR.

Reply:
I mistakenly thought these files were removed until you reminded me they're being copied over from node_modules. I'm not too worried about marginal gains so it's fine as is